groups/bsl/bslstl/bslstl_optional.h

```diff
@@ -217,26 +217,23 @@ BSLS_KEYWORD_INLINE_CONSTEXPR Optional_OptNoSuchType optNoSuchType =
 #else
 extern const Optional_OptNoSuchType optNoSuchType;
 #endif
-    // Value of type 'Optional_OptNoSuchType' used as the default argument in
-    // functions that are constrained using a function argument.
+// Value of type 'Optional_OptNoSuchType' used as the default argument in
+// functions that are constrained using a function argument.
 
 #ifdef BSLS_LIBRARYFEATURES_HAS_CPP11_BASELINE_LIBRARY
 
-template <class U, class V, bool D>
-struct Optional_IsConstructible : std::is_constructible<U, V> {
-    // This metafunction is derived from 'std::is_constructible<U, V>' in
-    // C++11 and later and from 'bsl::integral_constant<D>' in C++03 with the
-    // value of 'D' typically chosen to not affect the constraint this
-    // metafunction appears in.
-};
+// This macro is defined as 'std::is_constructible<U, V>' in C++11 and later,
+// and as 'bsl::integral_constant<DEFAULT>' in C++03 with the value of
+// 'DEFAULT' typically chosen to not affect the constraint this macro appears
+// in.
```

Comment:
Please update this to match the definitions

Reply:
done